Add support for default shank ix discriminators in nodes-from-anchor (#590)
codama-idl/codama: packages/nodes-from-anchor/README.md

pnpm install @codama/nodes-from-anchor
```

> > ⚠️ NOTES ⚠️
> >
> > - This package is **not** included in the main [`codama`](../library) package.
> > - A fair word of advice is that in case the `metadata.origin` key of the idl is not set, it is assumed to be `"anchor"`. So if you are trying to parse a Shank IDL, be sure that origin is set to `"shank"`.

Reviewer: nit: `[!NOTE]` is a special MD notation that displays the message is a special colour on GitHub so best to keep it that way.

```suggestion
> [!NOTE]
>
> - This package is **not** included in the main [`codama`](../library) package.
> - If `metadata.origin` is not set on the IDL, it is assumed to be `"anchor"`. If you are trying to parse a Shank IDL, be sure that origin is set to `"shank"` so discriminators can be set correctly.
```
